packages/kit/package.json

```diff
@@ -1,6 +1,11 @@
 {
 	"name": "@sveltejs/kit",
 	"version": "1.0.0-next.166",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/sveltejs/kit"
+	},
```

Comment:
We should also set `repository.directory` since this is a monorepo, so that relative links in a README are properly resolved. [Example](https://github.com/sveltejs/vite-plugin-svelte/blob/786c180f851b12042f53daef3c2bb2de4c744ad3/packages/vite-plugin-svelte/package.json#L35)

Reply:
Done!